includes/class-sensei-lesson.php

```diff
@@ -696,6 +696,16 @@ public function lesson_course_meta_box_content() {
 				// Add a course input fields
 				$html     .= '<div id="lesson-course-details" class="hidden">';
 					$html .= '<p>';
+
+						/**
+						 * Allows plugins to add additional content at the top of the course create form.
+						 *
+						 * @since 2.0.0
+						 *
+						 * @param string $html Current HTML for the course meta box as it currently is set up.
+						 */
+						$html = apply_filters( 'sensei_lesson_course_create_form_top', $html );
```

Comment:
I strongly disliked adding them as filters, but wasn't sure what our strategy was around refactoring non-WC specific code. Happy to do it though.

Reply:
But since these are newly added, couldn't we make them actions instead of filters, and use `ob_start()` and friends to get the resultant string and concatenate it? I'm not sure that would count as a refactor?